src/modules/job/dto/create-job.dto.ts

```diff
@@ -0,0 +1,21 @@
+import { IsNotEmpty, isString } from 'class-validator';
```

Comment:
Add documentation to your dto.for swagger..do this using the swagger decorators. For clarification reference the auth login and organisations module.